libs/gltfio/src/FFilamentAsset.h

```diff
@@ -255,9 +258,10 @@ struct FFilamentAsset : public FilamentAsset {
 
     void createAnimators();
 
-    filament::Engine* mEngine;
-    utils::NameComponentManager* mNameManager;
-    utils::EntityManager* mEntityManager;
+    filament::Engine* const mEngine;
+    utils::NameComponentManager* const mNameManager;
+    utils::EntityManager* const mEntityManager;
+    NodeManager* const mNodeManager;
```

Comment:
These could be references.

Reply:
Done.  (except for name manager because it's optional)